core-lib/Actors.ns

```diff
@@ -151,6 +152,14 @@ class Actors usingVmMirror: vmMirror usingKernel: kernel = Value (
     )
   )
 
+  private class Timer = Value (
```

Comment:
Why is this class necessary? Does it do anything?

Also, why is this a value?
Is it really appropriate to share timers between actors? This sounds like sharing race conditions between actors.